packages/ui/v2/core/Meta.tsx

```diff
@@ -0,0 +1,56 @@
+import Head from "next/head";
+import React, { createContext, useContext, useState } from "react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+
+type MetaType = {
```

Comment:
My worry with this PR is that the `MetaType` attributes are not typed so if an unsupported title/description is passed in it will use that string directly. And since the strings are snakecase it won't look good. 

For example for the conferencing page if `conferencing2` was passed in by accident, it would pass typing but not match supported within the i18n lookups. 
<img width="403" alt="Screen Shot 2022-08-30 at 12 53 06 PM" src="https://user-images.githubusercontent.com/6392501/187495910-6c0eb706-7f90-4073-b146-a73b6d17b37a.png">
<img width="237" alt="Screen Shot 2022-08-30 at 12 53 01 PM" src="https://user-images.githubusercontent.com/6392501/187495920-6025c623-6600-4d90-8db2-ed3dc8531c94.png">

This might be out of scope for this PR but we should look into typing i18n lookups.

We could also add a "Cal.com" backup for the title so it doesn't show up weird if the lookup fails.